### DISCUSSION TITLE
Minimum window width increased

### DISCUSSION THREAD

**Discussion by petti1589** [sumatrapdfreader/sumatrapdf#3109] [Q&A] [answered]
I like to make Sumatra pretty small to fit over a specific area in my handwriting program. Somehow the minimum width of Sumatra increased, so it is too big for my usecase.
I'm not sure if this is really Sumatra related or if it is just a Windows thing...

**Comment by GitHubRulesOK** [comment]
it increased over time for many reasons but shold not be drastic see https://github.com/sumatrapdfreader/sumatrapdf/issues/2425
related https://github.com/sumatrapdfreader/sumatrapdf/issues/3031 and https://github.com/sumatrapdfreader/sumatrapdf/discussions/2424

**Reply by petti1589** [reply]
Thanks for your answer. I tried updating Sumatra which did not fix it. After reinstalling the problem was gone.

Now a few days later the problem appeared again.
The minimum window width is now about 740 pixels. Before it has been below 560 pixels.
My screen is 1920 by 1080.

**Reply by petti1589** [reply]
I reinstalled again after saving the "faulty" settings file. With standard settings there is no problem. Reusing the old settings the problem reappeared. Deleting the settings file and starting Sumatra without a settings file also fixes my problem.
I tried lowering the Window state value but this did not work. Is there any other setting that could affect the window size?

**Reply by GitHubRulesOK** [reply]
unclear why the settings file is needing a rebuild to clear the problem, however one potential for that is the way the settings file is parsed may be different if it's with and without BOM (Byte Order) especially with notepad or WordPad save as

**Reply by petti1589** [reply]
I used the stock text editor in Windows 10.
This time I closed Sumatra (I think after closing all tabs) and a few minutes later I opened another file and there was the problem again. No change in settings, no restart of the system in between. I tried some things to recreate the problem but I had no "luck" so far...

~~I dont know if another program could cause this issue but here are the programs I usually use in parallel: Xournal++, ExtraButtons (to put Sumatra always in foreground), a tool to remap pen buttons (and fancy zones from power toys, but it wasnt installed when the problem occured the first time)~~

**Reply by petti1589** [reply]
I just tried again and found the issue. It occures after closing the bookmarks. Looking into the settings I found that the value of SidebarDx is different to the stock settings. Setting this value to 0 again solves the problem. ~~I also discovered that this value adds up when reopening and closing the bookmarks so the minimum width becomes really big...~~

**Comment by petti1589** [comment, marked as answer]
Short Answer:
Set the Value of **SidebarDx** in the settings text file to **0**. 

How this happens:
By closing the bookmark menu. If you dont need bookmarks you can also hide them as default.

**Reply by GitHubRulesOK** [reply]
@kjk I can confirm that if bookmarks were in a wide state `SidebarDx = 628` when session is closed any other session opened without bookmarks is forced wider than normal.
Unsure what can be done to reduce Dx on session open if bookmarks are not needed, since it was users choice from past session. Reset to minimum is likely to raise issues from those that want it wide on opening !

**Reply by hilmiwicak** [reply]
It doesn't work at all for me. I've changed every **SidebarDx** inside settings file, and it doesn't work at all after I've saved the file. It also doesn't work after I close sumatra and reopen it again, even sumatra changes the value again into 760

**Reply by GitHubRulesOK** [reply]
Try setting to 150 which is minimum but it must be done **before or after** SumatraPDF is running

**Reply by hilmiwicak** [reply]
@GitHubRulesOK thank you I just tried it and it works!
